### ISSUE TITLE
'More actions' button role and name

### ISSUE THREAD

**Issue by LiLoDavis** [microsoft/accessibility-insights-web#2774]
## Describe the bug

In Accessibility Insights for Android, each failure instance has a button that reveals a menu with two options: **File issue** and **Copy failure details**. 
### Current
The button's current role is **menu item**.
The button's current name is "More actions for card X in rule Y" where X is the class name of the failure instance and Y is the name of the rule. 
### Expected
The button actually functions as a **menu button**. According to the ARIA design pattern for [menu button](https://www.w3.org/TR/wai-aria-practices/#menubutton), the correct role for a menu button is simply **button**.
The name would be more task oriented if it referred to a "failure instance" instead of a "card". For example, "More actions for failure instance android.widget.TexView in rule ColorContrast".

**Comment by ferBonnin** [comment]
reviewed this with Peter. Makes sense that, since we are not expanding the menu items to be individual buttons ever, to make this role= button. We will need to revisit this when/if we expand this options.

**Comment by RobGallo** [comment]
It turns out the underlying HTML generated by ai-android is in accordance with the ARIA spec. In addition, both JAWS and NVDA render the button as a "menu button," which is what we expect. The only slightly odd piece is UIA (in Windows) gives the control type as MenuItem; which is somewhat unexpected, but perhaps a design decision on the part of UIA developers.
Given the code is correct, and screen readers render the control as expected, no changes will be made to change the control type/role.
However, the name of the button will be changed.

**Comment by RobGallo** [comment]
Resolved in PR #3149

**Comment by LiLoDavis** [comment]
I agree with your decision to make not changes in the control type/role. Name change looks good.
